Basic/Pod/PP.pod

```diff
@@ -2041,8 +2041,8 @@ Only used if C<HandleBad =E<gt> 1>.
 =item GenericTypes
 
 An array reference. The array may contain any subset of the one-character
-strings `B', `S', `U', `L', `Q', `F' and `D', which specify which types
-your operation will accept. The meaning of each type is:
+strings `B', `S', `U', `L', `Q', `F', `D', 'C' and 'G' which specify which
```

Comment:
Better to delete the list of letters entirely, given they are then listed straight after. This will also reduce the doc effort for adding further types such as [complex] long double.